superset/db_engine_specs/base.py

```diff
@@ -1339,8 +1343,16 @@ class BasicParametersMixin:
         "drivername://user:password@host:port/dbname[?key=value&key=value...]"
     )
 
+    # query parameter to enable encryption in the database connection
+    # for Postgres this would be `{"sslmode": "verify-ca"}`, eg.
+    encryption_parameters: Dict[str, str] = {}
```

Comment:
do you feel that we will need to check/enforce that this is defined on the engine specs that use this mixin?

Reply:
Oh, good idea... maybe if the users passes `encryption=True` when configuring a database we should raise an exception if this is empty.